lib/smart_proxy_dynflow_core/log.rb

```diff
@@ -8,7 +8,7 @@ class Log < ::Logger
     class << self
       def instance
         if @logger.nil?
-          @logger = Logger.new log_file
+          @logger = self.new log_file
```

Comment:
Seems the log_file is being written to by webrick access log (see the `launcher.rb`) which breaks the whole thing. We need to pass something else to the webrick access log (it seems it accepts the logger) instance.